Src/WitsmlExplorer.Frontend/components/ContentViews/table/ColumnOptionsMenu.tsx

```diff
@@ -233,29 +242,22 @@ export const ColumnOptionsMenu = (props: {
             }}
           >
             Reset ordering
-          </ResetButton>
-          <ResetButton onClick={resizeColumns}>Reset sizing</ResetButton>
+          </Button>
+          <Button style={{ width: "125px" }} onClick={resizeColumns}>
+            Reset sizing
+          </Button>
         </ResetContainer>
       </StyledMenu>
     </>
   );
 };
 
-const StyledButton = styled(Button)<{ colors: Colors }>`
-  color: ${(props) => props.colors.infographic.primaryMossGreen};
-`;
-
 const OrderingRow = styled.div`
   display: grid;
   grid-template-columns: 20px 25px 25px 1fr;
   align-items: center;
 `;
 
-const OrderingButton = styled(Button)`
```

Comment:
Isn't it better to keep these to avoid inline styles and duplicate code?

Reply:
Good point, fixed!